elytron/src/test/java/org/wildfly/extension/elytron/ElytronMixedStabilitySubsystemParsingTestCase.java

```diff
@@ -49,10 +49,12 @@ private static void mockReadResourceWithValidSubsystemTestFilePaths() {
             private String readResource(String name) throws IOException {
                 String namespaceUri = ElytronSubsystemSchema.CURRENT.get(Stability.DEFAULT).getNamespace().getUri();
                 String version = namespaceUri.substring(namespaceUri.lastIndexOf(':') + 1);
-                if (!name.contains(version + ".xml")) {
-                    return ModelTestUtils.readResource(getClass(), name.replace("elytron", "legacy-elytron-subsystem"));
-                } else {
+                String previewNamespaceUri = ElytronSubsystemSchema.CURRENT.get(Stability.PREVIEW).getNamespace().getUri();
+                String previewVersion = previewNamespaceUri.substring(previewNamespaceUri.lastIndexOf(':') + 1);
+                if (name.contains(version + ".xml") || name.contains(previewVersion + ".xml")) {
                     return ModelTestUtils.readResource(getClass(), name.replace("elytron", "elytron-subsystem"));
+                } else {
+                    return ModelTestUtils.readResource(getClass(), name.replace("elytron", "legacy-elytron-subsystem"));
                 }
             }
```

Comment:
I don't understand the purpose of mockReadResourceWithValidSubsystemTestFilePaths().  If the goal is simply to override the default file names of the subsystem test files, would it not be far simpler to just override the method used to generate them?
e.g.
```java
@Override
protected String getSubsystemXmlPathPattern() {
    // Same as super impl, but with "-subsystem-" thrown in
    String pattern = (this.schema.getStability() == Stability.DEFAULT) ? "%1$s-subsystem-%2$d.%3$d.xml" : "%1$s-subsystem-%4$s-%2$d.%3$d.xml";
    // If not a current schema, prefix with "legacy-"
    return !ElytronSubsystemSchema.CURRENT.values().contains(this.schema) ? "legacy-" + pattern : pattern;
}
```